src/platform/forms-system/src/js/review/submit-states/ErrorLinks.jsx

```diff
@@ -69,21 +69,19 @@ const ErrorLinks = props => {
                 {errors.map(error => (
                   <li key={error.name}>
                     {error.chapterKey ? (
-                      <a
+                      <va-button
                         href="#"
                         onClick={event => {
                           event.preventDefault();
-                          props.openReviewChapter(error.chapterKey);
```

Comment:
This work depends on being able to programmatically open an accordion. We should update it to click on the targeted accordion.

Reply:
Good call. I'm going on code here as nothing I've tried is able to generate these links. The best I can do is a generic "something went wrong, call us" error message. Do you have instructions on how to get them to show up? It appears I need to give them more attention.

Reply:
The [review errors docs are here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/docs/reviewErrors.md); they are currently implemented in the 526 (all-claims) form.

Reply:
The easiest way to get the error links to show up is to start a new form 526, then use the save-in-progress menu to jump to the review & submit page, then submit the form

Reply:
Ah thanks, that's a lot easier than the solution I came up with :)))